test/e2e/networks/ci.toml

```diff
@@ -34,6 +34,7 @@ perturb = ["restart"]
 [node.validator01]
 perturb = ["disconnect"]
 seeds = ["seed01"]
+block_sync = "v0"
```

Comment:
Can't we just remove this then? Also does the generator need to be updated

Reply:
done